docs/csharp/programming-guide/classes-and-structs/access-modifiers.md

```diff
@@ -18,6 +18,17 @@ All types and type members have an accessibility level. The accessibility level
 - [protected internal](../../language-reference/keywords/protected-internal.md): The type or member can be accessed by any code in the assembly in which it's declared, or from within a derived `class` in another assembly.
 - [private protected](../../language-reference/keywords/private-protected.md): The type or member can be accessed only within its declaring assembly, by code in the same `class` or in a type that is derived from that `class`.
 
+**Summary Table**
+Caller's location ↓  | public | protected internal | protected | internal | private protected | private 
+-- | -- | -- | -- | -- | -- | --
+**<ins>Within the assembly</ins>** |   |   |   |   |   |  
+Within the class | ✔ | ✔ | ✔ | ✔ | ✔ | ✔
+Derived class | ✔ | ✔ | ✔ | ✔ | ✔ | ❌
+Non-derived class | ✔ | ✔ | ❌ | ✔ | ❌ | ❌
+**<ins>In an external assembly</ins>** |   |   |   |   |   |  
```

Comment:
I'm also wondering why the `ins` tag is used?

Reply:
@BillWagner I'm using the `ins` tag to have an underline, but it seems to be causing issues. Would you say replacing the underline with italics still make the table readable?

Reply:
@DecimalTurn  Yes, let's use italics, or bold italic.